### PR TITLE
10039 to test

### PR DESCRIPTION
**Pull request by TomElliottFlexion**

### CODE REVIEW

File: scripts/judgeUpdates/update-senior-judges.ts [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+  for (let judge of seniorJudges) {
+    const { userId } = judge;
+
+    const userToUpdate = await applicationContext
+      .getPersistenceGateway()
+      .getUserById({ applicationContext, userId });
+    const userEntity = new User(userToUpdate);
+    userEntity.judgeTitle = 'Senior Judge';
+
+    console.log('userToUpdate', userToUpdate);
```
Comment (TomElliottFlexion) [inline scripts/judgeUpdates/update-senior-judges.ts]: Leaving this intentionally?

File: shared/src/business/useCases/judgeActivityReport/getCasesByStatusAndByJudgeInteractor.ts [hunk 2]
```diff
@@ -61,32 +57,38 @@ export const getCasesByStatusAndByJudgeInteractor = async (
...
-  const allCaseResults: CavAndSubmittedFilteredCasesType[] = caseRecords.map(
+  let allCaseResults: CavAndSubmittedFilteredCasesType[] = caseRecords.map(
     (caseRecord, i) => ({
       ...caseRecord,
       daysElapsedSinceLastStatusChange: daysElapsedSinceLastStatusChange[i],
       formattedCaseCount: numConsolidatedCases[i] || 1,
     }),
   );
 
+  allCaseResults = [
```
Comment (TomElliottFlexion) [inline shared/src/business/useCases/judgeActivityReport/getCasesByStatusAndByJudgeInteractor.ts]: Remove this

File: shared/src/business/useCases/judgeActivityReport/getCasesByStatusAndByJudgeInteractor.ts [hunk 3]
```diff
@@ -61,32 +57,38 @@ export const getCasesByStatusAndByJudgeInteractor = async (
...
-  let paginatedCaseResults;
-  if (searchEntity.pageSize && searchEntity.pageNumber) {
-    const itemOffset =
-      (searchEntity.pageNumber * searchEntity.pageSize) % allCaseResults.length;
-
-    const endOffset = itemOffset + searchEntity.pageSize;
-
-    paginatedCaseResults = allCaseResults.slice(itemOffset, endOffset);
-  }
+  allCaseResults.forEach(c => {
```
Comment (TomElliottFlexion) [inline shared/src/business/useCases/judgeActivityReport/getCasesByStatusAndByJudgeInteractor.ts]: ++

File: shared/src/business/useCases/judgeActivityReport/getCountOfCaseDocumentsFiledByJudgesInteractor.ts [hunk 4]
```diff
@@ -39,46 +31,23 @@ export const getCountOfCaseDocumentsFiledByJudgesInteractor = async (
...
+      applicationContext,
+      params: {
+        documentEventCodes: params.documentEventCodes,
+        endDate: searchEntity.endDate,
+        judges: searchEntity.judges,
+        startDate: searchEntity.startDate,
+      },
+    });
+
+  const computedEventCodes =
```
Comment (TomElliottFlexion) [inline shared/src/business/useCases/judgeActivityReport/getCountOfCaseDocumentsFiledByJudgesInteractor.ts:51]: Can eventCodes be returned by ES?

**Review by zachrog — APPROVED** [review]

**Pull request merged**